luno.go

```diff
@@ -14,26 +14,37 @@ import (
 	"runtime"
 	"strings"
 	"time"
+
+	"golang.org/x/time/rate"
 )
 
+type Limiter interface {
+	Wait(context.Context) error
+}
+
 // Client is a Luno API client.
 type Client struct {
 	httpClient   *http.Client
+	rateLimiter  Limiter
 	baseURL      string
 	apiKeyID     string
 	apiKeySecret string
 	debug        bool
 }
 
-const defaultBaseURL = "https://api.luno.com"
-
-const defaultTimeout = 10 * time.Second
+const (
+	defaultBaseURL = "https://api.luno.com"
+	defaultTimeout = 10 * time.Second
+	defaultRate    = time.Minute / 300
```

Comment:
I think it would be useful to add a link to the API documentation where this limit is mentioned.